ProjectManagmentSystem/src/Modules/Context/AuthContext.tsx

```diff
@@ -10,13 +10,15 @@ export const useAuth = () => {
 
 export default function AuthContextProvider(props: any) {
   const [loginUser, setLoginUser] = useState(null);
-
-
+  let requestHeaders = {Authorization:`Bearer ${localStorage.getItem('token')}`}
+  let baseUrl = 'https://upskilling-egypt.com:3003/api/v1'
 
   const getUserData = () => {
     const encodedToken: any = localStorage.getItem("token");
     const decodedToken: any = jwtDecode(encodedToken);
     setLoginUser(decodedToken);
+    console.log(decodedToken);
```

Comment:
We should remove logs before merging to the main branch to avoid revealing any sensitive data.

Reply:
Done